.github/workflows/generate-binaries.yml

```
@@ -19,24 +19,25 @@ jobs:
         uses: actions/checkout@v4
       - name: Extract branch name
         shell: bash
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/} | sed -e 's#/#-#g')"
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2001:style:1:21: See if you can use ${variable//search/replace} instead

(shellcheck)